react/README.md

```diff
@@ -34,12 +34,28 @@ For example, the following works:
 <MaskedTextInput
   mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
   className="form-control"
-  placeholder="Enter zip code"
+  placeholder="Enter a phone number"
   guide={false}
   id="my-input-id"
 />
 ```
 
+You can additionally use your own custom input component by passing in the prop `inputElement`.
```

Comment:
Ah good point, need to figure that out actually. Might need to use `React.cloneElement` here if a custom component gets passed in. But that won't work for stateless components.

I'll need to think about this a little more then.